vite.config.ts

```diff
@@ -7,7 +7,7 @@ export default defineConfig((configEnv) => ({
   build: {
     lib: {
       formats: ["es", "umd"],
-      entry: resolve(__dirname, "src/react-to-webcomponent.ts"),
+      entry: resolve(__dirname, "src/index/index.ts"),
```

Comment:
That file can just be at src/index.ts. It'll never have anything but those exports, so it doesn't need to be a modlet.